### PR TITLE
Make the gc sweep test run multiple times

### PR DESCRIPTION
**Pull request by tyler-cai-microsoft**
[AB#1719](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/1719)

This makes the GC Sweep test run 10 times.

Following PRs will fix several other smaller issues.

### CODE REVIEW

File: packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts [hunk 1]
```diff
@@ -86,27 +86,33 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
...
+        seed = Math.random();
+        random = makeRandom(seed);
+    });
+
     // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
     // Note: can run with npm run test:build to build and run the test
     // TODO: setup test to run in CI
+    const numberOfTests = 10;
+    for (let i = 0; i < numberOfTests; i++) {
     it(`GC Randomization Test with Seed: ${seed}`, async () => {
```
Comment (tyler-cai-microsoft) [inline packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts:106]: Note: whitespace was purposely not added so the PR was more readable

Comment (markfields) [reply packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts:100]: Will you have a follow-on that fixes the indentation?

File: packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts [hunk 2]
```diff
@@ -86,27 +86,27 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
...
-        return random.integer(min, max);
-    };
 
     // Time spent to run the test. Currently 10 seconds, for better coverage increase this number.
     const testTime = 10 * 1000;
 
     // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
     // Note: can run with npm run test:build to build and run the test
     // TODO: setup test to run in CI
+    const numberOfTests = 10;
```
Comment (markfields) [inline packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts:96]: Could be cool to have this configurable via Mocha cmd arguments.  For a later PR.

File: packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts (hunk 1 above)
Comment (tyler-cai-microsoft) [reply packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts:100]: I was thinking I would just add the spaces right before merging

**Review by markfields — APPROVED** [review]

**Pull request merged**